contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "../interfaces/OCR2DRClientInterface.sol";
+import "../interfaces/OCR2DROracleInterface.sol";
+
+/**
+ * @title OCR2DR oracle contract (stub for now)
+ */
+contract OCR2DROracle is OCR2DROracleInterface {
+  event OracleRequest(address sender, uint256 nonce, bytes data);
+  event CancelOracleRequest(bytes32 indexed requestId);
+  event OracleResponse(bytes32 indexed requestId);
+
+  function sendRequest(
+    address sender,
+    uint256 nonce,
+    uint256, /* subscriptionId */
+    bytes calldata data
+  ) external override {
+    emit OracleRequest(sender, nonce, data);
+  }
+
+  function cancelRequest(bytes32 requestId) external override {
+    emit CancelOracleRequest(requestId);
```

Comment:
We have to verify the caller right? Otherwise a user could cancel somebody else's request.

Reply:
Correct, we have to do many different things in Oracle. This Oracle is just a stub for now (see the top comment), because the purpose of this work is to deliver the client, not the oracle.